### PR TITLE
create L block with tests

### PR DESCRIPTION
**Pull request by DzmitryTatarski**

### CODE REVIEW

**Comment by marcin-ogorzalek-EPAM** [comment]
Already there are some merge conflicts to solve.

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,12 @@
             <version>7.4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.4.0</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:29]: Correct me if I'm wrong but this one is a duplicate. Moreover testing library definitely should stay in the `test` scope.

File: pom.xml [hunk 2]
```diff
@@ -21,6 +21,12 @@
             <version>7.4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.4.0</version>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:27]: I would like to see versions of the dependencies and plugins as in the `properties`. Use the variables to define those and use them in proper places.

File: src/main/java/com/epam/prejap/tetris/block/LBlockTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Block's shape testing: number of rows and cols, dots locations.
+ * @see LBlock
+ * @author Dzmitry_Tatarski
+ * @version 1.0
+ */
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/tetris/block/LBlockTest.java:13]: Well we rather avoid creating javadocs in the test classes

File: src/main/java/com/epam/prejap/tetris/block/LBlockTest.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+@Test(groups = { "blockShapes" })
+public class LBlockTest {
+
+    LBlock lBlock = new LBlock();
+
+    public void rows_ShouldReturn2() {
+        int expected = 3;
+        int actual = lBlock.rows();
+        assertEquals(actual, expected,
+                String.format("Number of rows is incorrect. Expected: " + expected + "actual:" + actual));
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/tetris/block/LBlockTest.java:23]: This is how it should be done with plain String not while we are using `String.format`. Please check out how to use variables with regard to that API.

File: src/main/java/com/epam/prejap/tetris/block/LBlockTest.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Block's shape testing: number of rows and cols, dots locations.
+ * @see LBlock
+ * @author Dzmitry_Tatarski
+ * @version 1.0
+ */
+@Test(groups = { "blockShapes" })
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/tetris/block/LBlockTest.java:14]: I like tests group here

**Review by marcin-ogorzalek-EPAM — CHANGES_REQUESTED** [review]
There are things to fix. Also resolve the merge conflicts and bump up the project version in `pom.xml`

**Comment by TJB4epam** [comment]
Conflicts reminder, please resolve.

**Comment by TJB4epam** [comment]
Conflicts and old remarks.

**Comment by TJB4epam** [comment]
This Wednesday we will focus on aggresively closing PRs. Just FYI.

**Pull request closed without merging**